CHANGELOG.md

```diff
@@ -1,17 +1,29 @@
 # Changelog
-All notable changes to KoalaBot will be documented in this file.
-A lot of these commands will only be available to administrators
+
+All notable changes to KoalaBot will be documented in this file. A lot of these commands will only be available to
+administrators
```

Comment:
Not a useful change, please undo

Reply:
I think it auto-formatted everything before I pushed, will undo